### DISCUSSION TITLE
Panopticon Connector version

### DISCUSSION THREAD

**Discussion by Razzo1987** [akeeba/panopticon#257] [Q&A (Community Support)] [answered]
On the page of the website there is a wronng version:
![image](https://github.com/akeeba/panopticon/assets/906604/09c2a8c1-1358-4b7d-89a6-4b58bbfe46a1)

Instead  of:
![image](https://github.com/akeeba/panopticon/assets/906604/5fca559f-1421-4c99-842c-e625bae37da6)

Joomla 4.4.0

**Comment by nikosdion** [comment, marked as answer]
Please install connector version 1.0.1 which addresses this issue.
